### PR TITLE
fix(ui): Don't show project picker for alert wizard v3

### PR DESCRIPTION
**Pull request by taylangocmen**
Followup to #33548

### CODE REVIEW

File: static/app/views/alerts/list/index.tsx [hunk 1]
```diff
@@ -123,6 +123,12 @@ class IncidentsList extends AsyncComponent<Props, State & AsyncComponent['state'
     this.setState({hasAlertRule, firstVisitShown, loading: false});
   }
 
+  get projectsFromIncidents() {
+    const {incidentList} = this.state;
+
+    return Array.from(new Set(flatten(incidentList?.map(({projects}) => projects))));
```
Comment (scttcper) [inline static/app/views/alerts/list/index.tsx:129]: use more es6 syntax
```suggestion
    return [...new Set(incidentList?.map(({projects}) => projects).flat())]
```

File: static/app/views/alerts/rules/index.tsx [hunk 2]
```diff
@@ -59,6 +59,12 @@ class AlertRulesList extends AsyncComponent<Props, State & AsyncComponent['state
     ];
   }
 
+  get projectsFromIncidents() {
+    const {ruleList = []} = this.state;
+
+    return Array.from(new Set(flatten(ruleList?.map(({projects}) => projects))));
```
Comment (scttcper) [inline static/app/views/alerts/rules/index.tsx:65]: same here

**Review by scttcper — APPROVED** [review]
just nit picking on syntax

File: static/app/views/alerts/rules/index.tsx (hunk 2 above)
Comment (taylangocmen) [reply static/app/views/alerts/rules/index.tsx]: 👍

File: static/app/views/alerts/list/index.tsx (hunk 1 above)
Comment (taylangocmen) [reply static/app/views/alerts/list/index.tsx]: 👍

**Pull request merged**